nixos/modules/services/misc/klipper.nix

```diff
@@ -69,6 +67,26 @@ in
           for supported values.
         '';
       };
+
+      firmware = mkOption {
+        description = "Firmwares klipper should manage";
+        default = { };
+        type = with types; attrsOf
+          (submodule {
+            options = {
+              enableFlashing = mkEnableOption ''
+                automatic flashing of firmware to microcontroller
+
+                WARNING: Be careful with this option. Enabling this will automatically flash your microcontroller on e.g. nixos-rebuild.
+                This can potentially brick your microcontroller. Alternatively, use `klipper-flash-$mcu` to flash manually.
+              '';
```

Comment:
```suggestion
              flashing.enable = mkEnableOption ''
                automatic flashing of firmware to microcontroller

                WARNING: Be careful with this option. Enabling this will automatically flash your microcontroller on e.g. nixos-rebuild.
                This can potentially brick your microcontroller. Alternatively, use `klipper-flash-$mcu` to flash manually.
              '';
```

Reply:
Did not commit this suggestion as this is used to determine if firmware should be flashing and needs renaming as well. Done in eea26e33dd01cd2a448112a4054b2ba932973036